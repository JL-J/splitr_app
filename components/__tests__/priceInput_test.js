import React from 'react';
import { TextInput } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import PriceInput from '../priceInput';

describe('<PriceInput />', function() {
  it('renders the component correctly', () => {
    const snap = renderer.create(
      <PriceInput />).toJSON();
    expect(snap).toMatchSnapshot();
  });

  it('renders one text input component', () => {
    const wrapper = shallow(<PriceInput />);
    expect(wrapper.find(TextInput)).toHaveLength(1);
  });

});
