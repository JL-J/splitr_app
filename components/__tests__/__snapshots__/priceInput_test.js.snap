// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PriceInput /> renders the component correctly 1`] = `
<TextInput
  allowFontScaling={true}
  placeholder="Estimated cost"
  style={
    Object {
      "borderColor": "gray",
      "borderWidth": 1,
      "height": 40,
    }
  }
  underlineColorAndroid="transparent"
/>
`;
